$OpenBSD: patch-meson_build,v 1.1 2019/05/06 18:13:26 ajacoutot Exp $

XXX temporarily disable tests to unbreak with recent e-d-s
XXX remove this patch when updating to 3.32

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -172,7 +172,6 @@ subdir('contrib')
 subdir('data')
 subdir('src')
 subdir('po')
-subdir('tests')
 
 if enable_gtk_doc
   subdir('doc/reference')
